mk_verificator/tests/system/mirgation/config.yaml

```diff
@@ -0,0 +1,11 @@
+---
+admin_username: admin
```

Comment:
In my vision that's general data for scope of tests, and better create kinda system-config, which will be placed higher in hierarchy than this one. I wanna discuss it, please wait for my pull request with changes.